packages/techdocs-container/techdocs-core/src/core.py

```diff
@@ -90,4 +98,13 @@ def on_config(self, config):
         config["markdown_extensions"].append("markdown_inline_graphviz")
         config["markdown_extensions"].append("plantuml_markdown")
 
+        # merge config supplied by user in the mkdocs.yml
+        mdx_configs_keys = config["mdx_configs"].keys()
+        mdx_configs_override_keys = mdx_configs_override.keys()
+        for key in mdx_configs_override_keys:
+            if key in mdx_configs_keys:
+                default_config = config["mdx_configs"][key]
+                override_config = mdx_configs_override[key]
+                default_config.update(override_config)
```

Comment:
✨ 

Worth mentioning that the explicit `.keys()` on the dictionary is not required when iterating and with `in`.

So

```
mdx_configs_override_keys = mdx_configs_override.keys()
for key in mdx_configs_override_keys:
```

can be

```
for key in mdx_configs_override
```

And
```
mdx_configs_keys = config["mdx_configs"].keys()
if key in mdx_configs_keys:
```

can be

```
if key in config["mdx_configs"]
```

Reply:
Thanks for the tip. Fixed it